addons/bestja_estimation_reports/models.py

```diff
@@ -0,0 +1,223 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, fields, api, exceptions
+
+
+class Project(models.Model):
+    _inherit = 'bestja.project'
+
+    estimation_reports = fields.One2many('bestja.estimation_report', inverse_name='project')
+    enable_estimation_reports = fields.Boolean(string="Raporty szacunkowe do zbiórki żywności")
+
+    use_estimation_reports = fields.Boolean(
+        compute='_compute_use_estimation_reports',
+        compute_sudo=True,
+        search='search_use_estimation_reports',
+    )
+
+    @api.one
+    @api.depends(
+        'enable_estimation_reports',
+        'parent.enable_estimation_reports',
+        'parent.parent.enable_estimation_reports'
+    )
+    def _compute_use_estimation_reports(self):
+        self.use_estimation_reports = (
+            self.enable_estimation_reports or
+            self.parent.enable_estimation_reports or
+            self.parent.parent.enable_estimation_reports
+        )
+
+    def search_use_estimation_reports(self, operator, value):
+        return [
+            '|',  # noqa
+                ('enable_estimation_reports', operator, value),
+            '|',
+                ('parent.enable_estimation_reports', operator, value),
+                ('parent.parent.enable_estimation_reports', operator, value),
+        ]
+
+
+class EstimationReportEntry(models.Model):
+    _name = 'bestja.estimation_report_entry'
+
+    estimation_report = fields.Many2one(
+        'bestja.estimation_report',
+        ondelete='cascade',
+        required=True,
+        string="raport szacunkowy",
+    )
+    day_in_store = fields.Many2one('bestja_stores.day', required=True, string="dzień")
+    day = fields.Date(
+        related='day_in_store.date',
+        store=True,
+    )
+    tonnage = fields.Float(required=True, string="Liczba kg")
+    store = fields.Many2one(
+        required=True,
+        related='day_in_store.store.store',
+        store=True,
+        string="sklep",
+    )
+    store_project = fields.Many2one(
+        'bestja.project',
+        required=True,
+        related='estimation_report.project'
+    )
+    store_name = fields.Char(
+        string="nazwa sklepu",
+        related='day_in_store.store.store.display_name',
+        store=True,
+    )
+    store_address = fields.Char(
+        required=True,
+        related='store.address',
+    )
+    store_city = fields.Char(required=True, related='store.city')
+    chain = fields.Many2one(
+        related='store.chain',
+        store=True,
+    )
+    organization = fields.Many2one(
+        'organization',
+        string="organizacja partnerska",
+        related='estimation_report.organization',
+        store=True,
+    )
+    responsible_organization = fields.Many2one(
+        'organization',
+        string="odpowiedzialna organizacja",
```

Comment:
Since it is a relational field, I guess that the description should be rather „organizacja odpowiedzialna”.